src/customJs/socialShareTool/SocialShareTool.js

```diff
@@ -3,6 +3,10 @@ import PropTypes from 'prop-types';
 
 export const SocialShareTool = (props) => {
   const values = props.values;
+  const { social_share_size: size } = props.values;
+  const { locale } = window['unlayer-extensions-configuration'];
+  const baseUrl = 'https://app2.dopplerfiles.com/MSEditor/images';
+  const sufixLang = locale === 'en' ? '_en' : '';
```

Comment:
No, I am talking about something like this:

```suggestion
  const baseUrl = 'https://app2.dopplerfiles.com/MSEditor/images';
  const sufixLang = intl.formatMessage('share_image_langsufix');
```

or better yet:

```suggestion
  const baseUrl = 'https://app2.dopplerfiles.com/';
  // Use it directly in the jsx
  const faceboockImageUrl = intl.formatMessage('share_image_facebook', { baseUrl });
```